src/libraries/System.Data.OleDb/src/OleDbTransaction.cs

```diff
@@ -137,15 +137,15 @@ internal OleDbTransaction(OleDbConnection connection, OleDbTransaction? transact
             _isolationLevel = isolevel;
         }
 
-        public new OleDbConnection Connection
+        public new OleDbConnection? Connection
```

Comment:
This is documented that could return null https://docs.microsoft.com/en-us/dotnet/api/system.data.oledb.oledbtransaction.connection?view=dotnet-plat-ext-3.1

Reply:
Yeah, the Connection property on all DbTransaction classes should indeed be nullable, thanks.